### PR TITLE
[Relay] Show yolo detection result in text.

### PR DESCRIPTION
**Pull request by huajsj**
Issue:
Current yolo detection test only provide a image drawing solution
to output detection result, but for console user, such result
would not available for a view.

Solution:
Here we add a text show function to show detection result in
text format for testing in console scenario.

### CODE REVIEW

**Comment by huajsj** [comment]
Hi @siju-samuel, could you help for review? thanks.

**Comment by siju-samuel** [comment]
@huajsj Thanks for the PR, could you please rebase this PR?

File: python/tvm/relay/testing/yolo_detection.py [hunk 1]
```diff
@@ -196,41 +196,91 @@ def do_nms_sort(dets, classes, thresh):
...
+                detection["bot"],
+                detection["width"],
+                rgb[0],
+                rgb[1],
+                rgb[2],
+            )
+            _draw_label(im, detection["top"] + detection["width"], detection["left"], label, rgb)
+
+
+def show_detections(im, dets, thresh, names, classes):
```
[the remark this answers is not in the document]
Comment (huajsj) [reply python/tvm/relay/testing/yolo_detection.py:270]: fixed.

File: python/tvm/relay/testing/yolo_detection.py [hunk 2]
```diff
@@ -196,41 +196,91 @@ def do_nms_sort(dets, classes, thresh):
...
+    "Draw the markings around the detected region"
```
[the remark this answers is not in the document]
Comment (huajsj) [reply python/tvm/relay/testing/yolo_detection.py]: fixed.

**Comment by huajsj** [comment]
Hi @siju-samuel , Thanks for the follow up, all comments get addressed.

Regards
Hua

**Review by siju-samuel — APPROVED** [review]
LGTM

**Comment by siju-samuel** [comment]
Thanks @huajsj This PR is merged.

**Pull request merged**